### PR TITLE
added jawsdb

### PR DESCRIPTION
**Pull request by thajedster**

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "password": null,
```
Comment (toshyamsundar) [inline config/config.json:18]: Don't you need a password?

Comment (thajedster) [reply config/config.json:18]: the guide didn't specify if i needed one.  is there a default password i need to include?

Comment (toshyamsundar) [reply config/config.json:18]: provide some password just to be on the safer side

Comment (thajedster) [reply config/config.json:18]: alright

**Pull request closed without merging**